src/data/complex/is_R_or_C.lean

```diff
@@ -601,6 +601,43 @@ ring_hom.map_finsupp_prod _ f g
 
 end is_R_or_C
 
+namespace finite_dimensional
+variables {K : Type*} [is_R_or_C K]
+
+open_locale classical
+open is_R_or_C
+
+/-- This instance generates a type-class problem with a metavariable `?m` that should satisfy
+`is_R_or_C ?m`. Since this can only be satisfied by `ℝ` or `ℂ`, this does not cause problems. -/
+library_note "is_R_or_C instance"
+
+/-- An `is_R_or_C` field is finite-dimensional over `ℝ`, since it is spanned by `{1, I}`. -/
+@[nolint dangerous_instance] instance is_R_or_C_to_real : finite_dimensional ℝ K :=
```

Comment:
Should `complex.finite_dimensional` be removed if this now exists? If not, it would be good to link to it in the documentation comment.

Reply:
This file currently has a ton of imports (it imports analysis.complex.basic which imports analysis.calculus.times_cont_diff, for largely unnecessary reasons), so it can't be used in many parts of the library, whereas `data.complex.module` is safe to import almost anywhere.  So -- not now, but maybe after a refactor!